### DISCUSSION TITLE
유경석 1회차 블로그 피드백 요청 드립니다 🙏🏻

### DISCUSSION THREAD

**Discussion by Yu-Kyeong-Seok** [Orm-camp/blog-feedback#4] [동글동글 2기 피드백]
https://ddong-e2.tistory.com/1

1차 프로젝트 회고글을 처음으로 작성한 글입니다.

부족한 부분이 많은 글이라, 적극적인 피드백 대환영입니다!

**Comment by juuyeoon** [comment]
경석님, 임주연 입니다! 회고글 작성하시느라 고생 많으셨습니다 🙌🏻
이슈사항에서 언급되었던 git 활용법, 프로젝트를 진행하시고 난 뒤에는 이해도가 조금 더 올라가셨나요?
만약 그렇다면 경석님과 같은 어려움을 겪고 있는 분들께 (같은 눈높이에서) 이해하신 내용들을 공유하시는 글을 작성해 보시는 건 어떨까 하는 생각도 듭니다.
프로젝트를 통해 깨닫게 된 것, 알게 된 것을 글로 정리하여 타인에게 나눌 정도가 된다면 정말 멋질 것 같네요!

**Comment by peppone-choi** [comment]
회고를 잘 쓰신 것 같습니다. 약한 딱딱한 문체를 좀 더 다듬으시면 더 쉽게 읽힐 것 같아요! 제가 할 말은 아니지만 (...) 수고 많으셨어요!

**Comment by ggaong** [comment]
같이 프로젝트 하면서 디자인부터 차근차근 잡아가보자고 해주셨던 경석님 덕분에 마크업과 CSS를 빨리 끝내 기능 구현에 좀 더 신경을 쓸 수 있었던 것 같습니다! 중간중간 마크업 관련 피드백도 주시고 역시 경력자,,👍
날씨 api 키값을 받고 포스트맨으로 날씨 불러왔을 때 에러가 떴다는 부분에서 좀 더 자세히 작성하면 나중에 같은 문제가 발생했을 때 참고하기 좋을 것 같아요!! 

프로젝트 고생 많으셨습니다😃

**Comment by hyyun1113** [comment]
경석님~ 블로그 글 잘읽었습니다! 
간결한듯 담백하게 글을 써주셔서 읽기 쉬웠습니다. 퍼블리셔에서 개발자로 성장하는 모습이 잘 보이는 블로그 글이었습니다. 
몇가지 제 의견을 추가드리자면 이번 프로젝트로 만든 결과물이 궁금합니다! 사진 또는 글로 구현결과가 있으면 좋을 것 같습니다. 
그리고 사용스킬에 API명을 쓰는 것 보다 구현내용에 어떤 API를 사용했는지등을 넣는건 어떠실까요?
퍼블리셔에서 앞으로 어떻게, 어떤 개발자로서 성장하는지 기대됩니다!

**Comment by dayoungh** [comment]
자바스크립트 활용부분에 부족함을 느끼신 부분, API를 사용해 본 경험에 대한 부분 등 개인적으로 경석님 글을 읽으면서 공감가는 부분이 많았기 때문에,, 더 많은 이야기를 듣고싶어요! 
저는 이슈사항에 대해 기록하지는 않았는데 경석님처럼 이 부분을 써두면 미래의 나에게 도움이 되겠다는 생각도 들었습니다 
좋은 글 감사합니다👏👏👏

**Comment by euneunsol** [comment]
경석님! 블로그 글 잘 읽었습니다! 
프로젝트 사용 스킬을 정리해 주셔서 어떤 기술들이 들어갔는지 한눈에 파악하기 쉬운것같아요!
좋은 글 감사합니다! :)
